src/components/wallet/TopCards/BalanceCard/BalanceCard.vue

```diff
@@ -227,26 +243,39 @@ export default class BalanceCard extends Vue {
         return ''
     }
 
+    // Locked balance is the sum of locked CAM tokens P chain (bonded + deposited + bondedAndDeposited)
     // Locked balance is the sum of locked AVAX tokens on X and P chain
```

Comment:
Remove old description

Reply:
both descriptions are valid, we still support the avalanche way, and also our method of calculating the total balance